simulate_presence_script.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/python3
+
+import requests
+import json
+import random
+seat_seat = 'http://127.0.0.1:8000/api/seats/'
+halls_url = 'http://127.0.0.1:8000/api/halls/'
+payload = {}
+headers = {'Authorization': 'Bearer MWZiYmM5YjE3MWEwNzM0MzlmMjQwYjI0ZDE5MjMzZDMxZDA3YzcyOA=='}
+
+response_seats = requests.request("GET", seat_seat, headers=headers, data=payload)
+response_halls = requests.request('GET', halls_url, headers=headers, data=payload)
```

Comment:
All the request methods also have a "shortcut" method defined to shorted the length of the lines; requests.request("GET", URL) -> requests.get(URL).
If you prefer it this way it's no problem, I just wasn't sure you were aware these methods exist.